dffml/service/dev.py

```diff
@@ -376,7 +376,7 @@ class SetupPy(CMD):
     kwarg = SetupPyKWArg
 
 
-class RepoDirtyError(Exception):
+class RecordDirtyError(Exception):
```

Comment:
This one is talking about a git repo

```suggestion
class RepoDirtyError(Exception):
```